### ISSUE TITLE
Help! xpi can be install since addon is not been verified

### ISSUE THREAD

**Issue by piyarad27** [gorhill/uBlock-for-firefox-legacy#294]
- [x] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [x] uBlock Origin is the only extension
    - [x] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

[Description of the bug or feature]

I try to install uBlock0_1.16.4.28.firefox-legacy.xpi file and get the warning of "add-on could not be install because it had not been verified" on my FF browser this include  uBlock0_1.33.2.firefox.signed.xpi since it say that it not compatible with FF 53.0.3 

[A specific URL is MANDATORY for issue happening on a web page, even if it happens "everywhere"]

### Steps to Reproduce

1. download uBlock0_1.16.4.28.firefox-legacy.xpi/  uBlock0_1.33.2.firefox.signed.xpi
2. set xpinstall.signatures.required to false
3. go to about:addons > click the gear icon > select [install add-on from file] > select [uBlock0_1.16.4.28.firefox-legacy.xpi]
OR
drag and drop the xpi file into the browser

### Expected behavior:

the add-on is install without any warning

### Actual behavior:

get cannot be verified warning even xpinstall.signatures.required set to false

### Your environment

* uBlock Origin version:  uBlock0_1.16.4.28.firefox-legacy.xpi/  uBlock0_1.33.2.firefox.signed.xpi
* Browser Name and version: firefox 53.0.3 and 55.0.3
* Operating System and version:
![Screenshot (964)](https://user-images.githubusercontent.com/58059329/108585103-4c749a80-72fb-11eb-8181-157c4cd736a3.png)

**Comment by gwarser** [comment]
https://stackoverflow.com/questions/31952727/how-can-i-disable-signature-checking-for-firefox-add-ons#answer-42403531

**Comment by piyarad27** [comment]
I try the step on 
https://www.ghacks.net/2016/08/14/override-firefox-add-on-signing-requirement/
but it not work

**Comment by Vangelis66** [comment]
@piyarad27 

Please read [this](https://github.com/JustOff/ublock0-updater/issues/4#issuecomment-338329502);
Firefox **53.0.3** stable
https://web.archive.org/web/20181010094852/http://www.largrizzly.net:80/firefox.html#pre55
Firefox **55.0.3** stable
https://web.archive.org/web/20181010094852/http://www.largrizzly.net:80/firefox.html#55+

(thankfully, the **web archive** has also captured the links to the zip files... 👍 )

**Comment by gwarser** [comment]
> I try the step on
> [ghacks.net/2016/08/14/override-firefox-add-on-signing-requirement](https://www.ghacks.net/2016/08/14/override-firefox-add-on-signing-requirement/)
> but it not work

This is the place where I learn this is possible. I tried solution from my link and it worked on 53. 55 did not rendered GitHub pages correctly.

**Comment by Vangelis66** [comment]
> 55 did not rendered GitHub pages correctly.

M$ owned GitHub no longer supports **Fx < 60**; and to get GitHub working in **Fx 60esr**, you have to enable some about:config prefs related to **webcomponents**; M$ have fully succumbed to Google and now demand in their **list of sanctioned browsers** that they support **Google-exclusive web technologies** like Custom Elements, ShadowDOM, etc. 😠 

You can get ca. **90%** of GitHub functionality back in Fx 52/53, possibly even in 54/55, if you force-install @JustOff 's **[github-wc-polyfill](https://github.com/JustOff/github-wc-polyfill)**; NB, installation on **Fx** is NOT supported by its maintainer! For **Fx 55.0**, perhaps the WE **Polly** by zakius could be used; that is _if you're able to get your hands on it now_, as the repo originally hosting it has been vacuumed... 👎 
https://web.archive.org/web/20210215211345/https://github.com/zakius/Polly/

**Comment by piyarad27** [comment]
I still get add-on cannot be verified error even I try the step in
Firefox 55.0.3 stable
https://web.archive.org/web/20181010094852/http://www.largrizzly.net:80/firefox.html#55+
even you say that "1.33.x requires at minimum Firefox 55.0"

![Screenshot (28)](https://user-images.githubusercontent.com/58059329/108636110-1852c400-7438-11eb-9b7e-0863dd3a216d.png)

**Comment by Vangelis66** [comment]
> I still get 
> **add-on cannot be verified error** 

Read more carefully; the error you get is not **signature** related, but **appVersion** related: 

> uBlock Origin could not be installed because **it is not compatible with Firefox 55.0.3**

> even I try the step in Firefox **55.0.3** stable

Actually, I was **wrong**, so shoot me for trying to help... 
You only had to extract the **1.33.2 xpi** file 
and inspect **manifest.json** to discover that: 
```json
  "browser_specific_settings": {
    "gecko": {
      "id": "uBlock0@raymondhill.net",
      "strict_min_version": "57.0"
    }
```
IOW, **1.33.2** needs at least Fx **57**.0...
Alternatively, visit [AMO](https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/versions/) and **get yourself** the answers you seek: 
```
Version 1.33.2
Released Feb 1, 2021 - 2.62 MB
Works with firefox 57.0 and later, android 57.0 and later
```
and further below: 

```
Version 1.32.4
Released Dec 29, 2020 - 2.62 MB
Works with firefox 55.0 and later, android 55.0 and later
```
As I stressed earlier, this is _not_ the place to ask for support for WE editions of uBlock Origin...
